### PR TITLE
Add mapi alibaba image promotion

### PR DESCRIPTION
**Pull request by alexander-demicev**
This PR adds image promotion for https://github.com/openshift/cluster-api-provider-alibaba/

### CODE REVIEW

File: ci-operator/jobs/openshift/cluster-api-provider-alibaba/openshift-cluster-api-provider-alibaba-main-presubmits.yaml [hunk 1]
```diff
@@ -71,6 +71,7 @@ presubmits:
         - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
         - --report-credentials-file=/etc/report/credentials
         - --target=[images]
+        - --target=[release:latest]
```
Comment (JoelSpeed) [inline ci-operator/jobs/openshift/cluster-api-provider-alibaba/openshift-cluster-api-provider-alibaba-main-presubmits.yaml:74]: This seems to be causing an issue with the build, was this generated?

Comment (alexander-demichev) [reply ci-operator/jobs/openshift/cluster-api-provider-alibaba/openshift-cluster-api-provider-alibaba-main-presubmits.yaml:74]: yes, it was generated

**Pull request merged**